features/providers/partner_means_assessment/check_your_answers.feature

```diff
@@ -0,0 +1,21 @@
+Feature: partner_means_assessment check your answers
+  @javascript @vcr
+  Scenario: I am able to add a partner with an address shared with the applicant and then change it
+    Given the feature flag for partner_means_assessment is enabled
+    And I complete the journey as far as check client details with a partner
+
+    When I click Check Your Answers Change link for 'partner_address'
+    Then I should be on a page with title "Is the partner's correspondence address the same as your client's?"
+
+    When I choose "No"
+    And I click 'Save and continue'
+    Then I should be on a page with title "Enter the partner's correspondence address"
+    And the value of the "address lookup postcode field" input should not be "SW1H 9EA"
```

Comment:
Is this checking that the lookup field has been cleared correctly?

Reply:
It is 👍
When I first wrote the test it was failing so I wrote a test to make sure fixes didn't break anything else

Reply:
Yes.  When I first ran the tests, the values were being retained so I wrote this to ensure it had been cleared when I was changing the code